.templates/php/default.mk

```diff
@@ -0,0 +1,51 @@
+# Please update /.templates/php/default.mk and sync:
+#
+#     source scripts/functions.sh && rsync_files
+#
+SHELL := /usr/bin/env bash
+PHP_SOURCE_FILES = $(shell find . -name "*.php")
+
+### COMMON stuff for all platforms
+
+### Common targets for all functionalities implemented on php
+
+default: .tested
+.PHONY: default
+
+pre-release: update-version update-dependencies
+.PHONY: pre-release
+
+update-version:
+ifdef NEW_VERSION
+# TODO: something here?
```

Comment:
Nothing is required here; Composer takes its versioning from the release tags so nothing in the repo needs the version number

Reply:
Great
However the `update-version` target still needs to be there to be compatible with other makefiles